plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -89,7 +89,42 @@ func LeastRequestedPriority(pod *api.Pod, podLister algorithm.PodLister, minionL
 
 	list := algorithm.HostPriorityList{}
 	for _, node := range nodes.Items {
-		list = append(list, calculateOccupancy(pod, node, podsToMachines[node.Name]))
+		list = append(list, calculateResourceOccupancy(pod, node, podsToMachines[node.Name]))
+	}
+	return list, nil
+}
+
+func min(l, r int64) (m int64) {
+	m = r
+	if l < r {
+		m = l
+	}
+	return m
+}
+
+// See comment for DumbSpreadingPriority()
+const dumbSpreadingDenominator int64 = 10
+
+// DumbSpreadingPriority is a priority function that favors nodes with fewer pods.
+// It works like LeastRequestedPeriority but instead of using 10 * percentage of machine free by resource,
+// it uses 10 * percentage of machine free by pod, with "percentage of machine free by pod" claculated as
+// (dumbSpreadingDenominator - number of pods already on the node + 1) / dumbSpreadingDenominator.
+// dumbSpreadingDenominator serves like the machine capacity in LeasRequestedPriority but is chosen
+// so that we equate one pod with a reasonable amount of resources when we combine all the scores together.
```

Comment:
I'm not asking for dynamic weights, just for clarification around the following scenario.

Your score function:
```
(dumbSpreadingDenominator - number of pods already on the node + 1) / dumbSpreadingDenominator
```
When used with:
```
calculateScore(min(npods+1, dumbSpreadingDenominator), dumbSpreadingDenominator, node.Name)
```
Will result in the score:
```
int(((capacity - requested) * 10) / capacity) = 0
```
for npods >=10.

So if I start off with 4 nodes that have 2,2,2,1 addons (we currently have 7 addons, just to give you a realistic scenario), and create a 100 pod rc in a namespace without limits, will I end up with ~70 pods on the last node?

If the answer is yes, maybe we should acknowledge that in the docs. If the answer is no, please correct me :)